rfc/rfc023-x509credential.md

```diff
@@ -0,0 +1,453 @@
+  requirements in this document.
+- **did:x509**: A Decentralized Identifier (DID) method specified by the Trust Over IP Foundation, where the DID is
+  derived from an X.509 certificate.
+- **Issuer Certificate**: The X.509 certificate associated with the `did:x509` DID that issued the credential.
+- **Credential Subject**: The entity described by the credential.
+- **Revocation Check**: The process of verifying the revocation status of the issuer certificate using mechanisms like
+  OCSP or CRL.
+
+## Used standards and technologies
+
+This RFC builds on the following standards and technologies:
+
+* [X.509 Certificate Standard](https://datatracker.ietf.org/doc/html/rfc5280)
+* [JSON Web Signature (JWS)](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.6)
+* [did:x509 method specification](https://trustoverip.github.io/tswg-did-x509-method-specification/), with modifications
+* W3C Verifiable Credentials Data Model
+
+### x509 certificates, a brief introduction
+
+The structure of an x509 certificate is defined by the X.509 standard. An x509 certificate contains information about
+the holder and is signed by a Certificate Authority (CA). The CA is a trusted party that is capable of verifying the
+identity of the holder of the certificate. The CA signs the certificate with its own private key. The holder of the
+certificate can then use the public key of the CA to verify the signature of the certificate. This way the holder of the
+certificate can prove that the certificate is valid and that the information in the certificate is correct.
+
+The verifier of a x509 certificate can then trust the information in the certificate by verifying the signature of the
+certificate chain of the certificate. The verifier can then trust the information in the certificate by trusting the CA
+that signed the certificate.
+
+The chain of certificates can be viewed as a hierarchy, where the root certificate is the certificate is trusted, and
+signing is delegated to intermediate certificates. The root certificate is the certificate that is trusted by the
+holder. The holder maintains a list of trusted CAs that the holder trusts. The holder can then
+verify the signature of the certificate chain by verifying the signature of the CA that signed the intermediate
+certificate and the intermediate certificates that lead to the signing certificate.
+
+```asciidoc
+┌────────────────────┐
+│        CA          │
+└─────────┬──────────┘
+          │           
+┌─────────▼──────────┐
+│    Intermediate    │
+└─────────┬──────────┘
+          │           
+┌─────────▼──────────┐
+│    Intermediate    │
+└─────────┬──────────┘
+          │           
+┌─────────▼──────────┐
+│Signing Certificate │
+└────────────────────┘
+```
+
+### Using x509 for signing JWEs
+
+The JWE is a standard that is used to sign and encrypt JSON objects. Thus standard allows for the signing and encryption
```

Comment:
JWE is only for encryption (which is not applicable here), JWS is for signing. JWTs are a form of JWS', and since we only use JWTs, I think you can just call them JWTs in this section?